lib/bitcoin/BitcoinClient.ts

```diff
@@ -335,7 +350,7 @@ export default class BitcoinClient {
       method: 'getblockcount'
     };
 
-    const response = await this.rpcCall(request, true);
+    const response = await this.rpcCall(request, true, false);

```

Comment:
Style rant: just FYI, generally hate this kinda innocent looking call. Normally prefer `const isWalletRpc = false` then pass that in, else no clue what all these true false things are. But since there is already an existing precedence... will let it go!

Reply:
Will add it.